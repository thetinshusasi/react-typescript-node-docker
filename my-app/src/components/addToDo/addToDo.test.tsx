import React from "react";
import { shallow } from "enzyme";
import AddTodo from "./index";
import { Button, Form } from "react-bootstrap";

describe("<AddTodo />", () => {
  it("renders AddTodo", () => {
    const component = shallow(<AddTodo />);
    expect(component).toMatchSnapshot();
    expect(component.find(Form.Control).length).toEqual(2);
    expect(component.find(Button).length).toEqual(1);
    expect(component.find(Button).text()).toEqual("Add Task");
  });
});
