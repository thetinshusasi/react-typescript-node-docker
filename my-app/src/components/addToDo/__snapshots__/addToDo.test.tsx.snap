// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddTodo /> renders AddTodo 1`] = `
<Row
  noGutters={false}
>
  <Col
    className="m10px"
    sm={3}
    xs={12}
  >
    <FormControl
      name="title"
      onChange={[Function]}
      placeholder="Title"
      type="text"
      value=""
    />
  </Col>
  <Col
    className="m10px"
    sm={6}
    xs={12}
  >
    <FormControl
      name="description"
      onChange={[Function]}
      placeholder="description"
      type="text"
      value=""
    />
  </Col>
  <Col
    className="m10px"
    sm={3}
    xs={12}
  >
    <Button
      active={false}
      block={true}
      disabled={false}
      onClick={[Function]}
      variant="primary"
    >
      Add Task
    </Button>
  </Col>
</Row>
`;
